python/servo/post_build_commands.py

```diff
@@ -95,8 +95,18 @@ def run(self, params, release=False, dev=False, android=False, debug=False, debu
                 print("Could not find a suitable debugger in your PATH.")
                 return 1
 
+            command = self.debuggerInfo.path
+            if debugger == 'gdb' or debugger == 'lldb':
+                rustCommand = 'rust-' + debugger
+                try:
+                    result = subprocess.call([rustCommand, '--version'], env=env, stdout=open(os.devnull, 'w'))
```

Comment:
That would make it Unix-specific. I searched for this specific problem, and found this as the best solution. (In Python 3.3 there is a dedicated portable function IIRC, but not in 2.7.)

Another difference is that the way I implemented it now, it actually checks whether the command runs without error. I'm not sure whether this is really needed -- but it could be trivially changed to just ignore the result if preferred...